components/PackageOverview.js

```diff
@@ -111,49 +107,65 @@ export const PackageOverview = () => {
     [fileSearchTerm]
   );
 
+  if (!versions || !directory || !versions[request.version] || !directory.files)
```

Comment:
might be overkill but for some of the strings do we want to make sure they're not empty length too?

Reply:
These are all objects except for `directory.files` which is an array, so I think these checks are sufficient.